backend/app/services/ddah_service.rb

```diff
@@ -119,14 +119,7 @@ def normalize_duty(duty:)
     end
 
     def ddah_params
-        @params.slice(
-            :assignment_id,
-            :approved_date,
-            :accepted_date,
-            :revised_date,
-            :emailed_date,
-            :signature
```

Comment:
We don't want to permit these to be updated through the service, since individual routes implement functionality to accept, sign, and email ddahs.